org.mixedrealitytoolkit.uxcomponents.noncanvas/Slider/NonCanvasSliderBase.prefab

```diff
@@ -154,8 +154,8 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 2624031095332604921}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: 0, y: 0, z: 0.000000012922101}
-  m_LocalScale: {x: 0.099999905, y: 0.002, z: 1}
+  m_LocalPosition: {x: -9.313226e-10, y: 0, z: 0}
```

Comment:
why the position change?

Reply:
zero clue. i moved _nothing_ in the prefab. only added the AudioSouurce component. This looks like floating point error on the reserialization

Reply:
please revert

Reply:
i tried reverting and Unity 2021.3.21f` (matching version for the template project) reserialized this in as well.

the only thing i can think is to NOT make the prefab changes

Reply:
I'm able to repro these changes locally using Unity 2021.3.21f1 (down to the last decimal point, which is actually kinda impressive haha) by doing nothing except adding the AudioSource component.

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/3580640/670b649e-2047-4493-924f-ad78a37af2d9)

This seems like some Unity-internal serializer update, so I'm pretty sure it's unavoidable and we should take the changes now or we'll be plagued with them changing locally every time a change is made to these prefabs.

Reply:
resolving: per @keveleigh's repro and that this is a Unity 2021.3 reserialization, i believe we should accept this change so that these questions do not arise for any future change to this prefab